From 0edd6f5579c964323d86897aff9476c20d7901ba Mon Sep 17 00:00:00 2001
From: Vojtech Trefny <vtrefny@redhat.com>
Date: Mon, 22 Nov 2021 14:23:08 +0100
Subject: [PATCH] tests: Do not check that writeAmplificationRatio is bigger
 than 0

Apparently the bios_in_write value is now 0 for newly created VDO
pools and because we use it for the writeAmplificationRatio
calculation we get zero too.
---
 src/tests/dbus-tests/test_20_LVM.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/tests/dbus-tests/test_20_LVM.py b/src/tests/dbus-tests/test_20_LVM.py
index 7fbff0434..fde7c3f3f 100644
--- a/src/tests/dbus-tests/test_20_LVM.py
+++ b/src/tests/dbus-tests/test_20_LVM.py
@@ -508,7 +508,6 @@ def test_create(self):
         # get statistics and do some simple sanity check
         stats = lv.GetStatistics(self.no_options, dbus_interface=self.iface_prefix + '.VDOVolume')
         self.assertIn("writeAmplificationRatio", stats.keys())
-        self.assertGreater(float(stats["writeAmplificationRatio"]), 0)
 
     def test_enable_disable_compression_deduplication(self):
         vgname = 'udisks_test_vdo_vg'
